.gitignore

```diff
@@ -8,3 +8,5 @@
 **__pycache__/
 *.pyc
 **db.sqlite3
+*.jpg
+*.png
```

Comment:
There is an extra space before the `*.png` entry. Please remove this space to ensure that `.png` files are properly ignored by Git.